src/Compilers/Test/Utilities/VisualBasic/ParserTestUtilities.vb

```diff
@@ -80,7 +80,9 @@ Friend Module ParserTestUtilities
             Dim errors = PooledStringBuilderPool.Allocate()
             AppendSyntaxErrors(tree.GetDiagnostics(), errors.Builder)
             Assert.Equal(root.ContainsDiagnostics, errors.Builder.Length > 0)
-            Assert.False(root.ContainsDiagnostics, errors.ToStringAndFree())
+            Dim result = errors.ToStringAndFree
+            Assert.False(root.ContainsDiagnostics, result)
```

Comment:
`errors.ToStringAndFree()` will be invoked if execution reaches line 83, with or without this change.